pssh/pssh_client.py

```diff
@@ -40,17 +40,18 @@
 class ParallelSSHClient(object):
     """Uses :mod:`pssh.ssh_client.SSHClient`, performs tasks over SSH on multiple hosts in \
     parallel.
-    
+
     Connections to hosts are established in parallel when ``run_command`` is called,
     therefor any connection and/or authentication exceptions will happen on the
     call to ``run_command`` and need to be caught.
     """
-    
```

Comment:
These indentations are there purposefully as they are used when generating docstrings.

Please revert all indentation changes in this PR.